federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -22,7 +22,11 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
+        {{if eq .KUBERNETES_PROVIDER "aws"}}
+        - --external-hostname={{.FEDERATION_API_HOST}}
```

Comment:
IIRC, it's something to do with the automatic certificate generation logic and AWS loadbalancers giving your DNS names instead of IP addresses.